### ISSUE TITLE
Not able to access the frontend for meilisearch instance on GCP

### ISSUE THREAD

**Issue by tanmaymaheshwari97** [meilisearch/meilisearch#2435]
I have delpoyed a meilisearch instance on Google Cloud Platform as per the instructions in the official documentation. But unlike my local meilisearch instance running on localhost:7700, where I can see all the indexes along with a search bar on the browser, I am not able see frontend for the meilisearch instance deployed on GCP. Can someone please tell how to access the frontend on the browser.
<img width="836" alt="Screenshot 2022-05-27 at 10 39 14 AM" src="https://user-images.githubusercontent.com/66941891/170633720-e2ccac2e-5d65-4e70-857b-ddd5f654a9f6.png">

**Comment by sanders41** [comment]
Are you running Meilisearch in production mode on GCP? The frontend is only available in dev mode.

**Comment by tanmaymaheshwari97** [comment]
Yes, I am running it in production mode. Can you please tell where can I see the meilisearch logs?

**Comment by tanmaymaheshwari97** [comment]
> Are you running Meilisearch in production mode on GCP? The frontend is only available in dev mode.


Yes, I am running it in production mode. Can you please tell where can I see the meilisearch logs?

**Comment by sanders41** [comment]
I have not used GCP before so I am not sure where to find the logs. @alallema might know?

**Comment by alallema** [comment]
Hi @tanmaymaheshwari97,
As @sanders41 told you the dashboard is only accessible in development mode, you can change it by running the command `meilisearch-setup` if you use the GCP image from the documentation and then select `development` mode.
For the logs `Meilisearch` is running as a service so they will be available in this file `/var/log/syslog` or if you want the last hour of logs you can run this command:
```
journalctl --no-pager -u meilisearch.service
```
